### PR TITLE
Support custom git repo

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 monasca_checks: {}
 monasca_agent_system_only: False
 agent_reconfigure_script: /usr/local/bin/monasca-reconfigure
-
+default_monasca_agent_git_repo: https://git.openstack.org/openstack/monasca-agent
```
Comment (dougszumski) [inline defaults/main.yml]: I think you can just call this ```monasca_agent_git_repo```, set the default and then override the variable if you need to. No need for two variables.

Comment (brtkwr) [reply defaults/main.yml]: Is this better?

**Comment by brtkwr** [comment]
Is this better @dougszumski ?

File: defaults/main.yml (hunk 1 above)
Comment (dougszumski) [reply defaults/main.yml]: It works, although it's more canonical to put the default in the defaults/main file

**Review by dougszumski — APPROVED** [review]
Thanks @brtknr!

**Pull request merged**